kafkastats/src/main/java/com/pinterest/doctorkafka/util/OperatorUtil.java

```diff
@@ -87,12 +101,100 @@ public static boolean pingKafkaBroker(String host, int port, int timeout) {
       socket.connect(new InetSocketAddress(host, port), timeout);
       return true;
     } catch (UnknownHostException e) {
+      LOG.warn("Ping failure, host: " + host, e);
       return false;
     } catch (IOException e) {
+      LOG.warn("Ping failure IO, host: " + host, e);
       return false; // Either timeout or unreachable or failed DNS lookup.
     }
   }
 
+  public static boolean fetchData(String host, int port, String topic, int partition) {
```

Comment:
as this method return a boolean type, shall we name it `canFetchData`?

Reply:
Changed

Reply:
sorry if I missed something, but I still see `fetchData` as the method name...